### PR TITLE
[WIP] [RFC] Qt/GraphicsWindow: Add images to illustrate descriptions

### PR DESCRIPTION
**Pull request by spycrab**
This currently only adds images for two options (as a proof of concept):
* Show FPS
* Show NetPlay Ping

### Rationale

Illustrating some descriptions might help users to better understand what options do and make interacting with the graphics window more intuitive.

This is obviously not suited for every option but many of them could benefit from that.

### Screenshots

screen shot 2019-03-07 at 16 09 06

### CODE REVIEW

**Comment by MayImilae** [comment]
I'm not in favor of this, because this is just too limited and hurts consistency. Showing an FPS counter without context does't really give anything that "show number of frames per second on screen" doesn't provide, and there's no way to really add more usefulness to it due to the description area limitations. There are lots of other options where images could genuinely be useful, but the description area setup doesn't have enough room to allow for that. To properly take advantage of images would require a rethinking of how the menus are configured entirely, as well as providing lots of images and new descriptions. 

Anyway, TLDR, this is just not useful enough to warrant the loss in consistency, and there is no way for it to become more useful due to the window design.

**Comment by jordan-woyak** [comment]
I also think this is unnecessary and will require additional maintenance to keep the screenshots up to date as things change but for little benefit.

**Comment by spycrab** [comment]
> Showing an FPS counter without context does't really give anything that "show number of frames per second on screen" doesn't provide, and there's no way to really add more usefulness to it due to the description area limitations

Yeah it's not really useful for Show FPS per se. I just picked it as an easy to implement example (Same with NetPlay Ping).

> I also think this is unnecessary and will require additional maintenance to keep the screenshots up to date as things change but for little benefit.

I don't think this will be required at all though? Most of the things I'd show would be pretty unrelated to UI. For example something illustrating anti-aliasing doesn't need any updating at all as far as I can tell.

**Comment by JMC47** [comment]
I actually think this would work well for other options though, like Skip EFB Access to CPU and EFB2RAM, where we can directly show side by side how it affects certain games.  It'd give users a real world example to see how it works.  Imagine a screenshot cropped down a bit of Wind Waker's sun with it on/off.  And because it's not going to show any screenshots of the GUI, there isn't really any maintenance cost.

These on/off situations would work for everything on the hacks page, where there is a lot of misinformation already and I think it'd help.

**Comment by jordan-woyak** [comment]
I could see the usefulness of screenshots for AA settings and maybe a few others. But I don't know if we should have screenshots of the indirect effects of EFB settings in a random game.

**Comment by MayImilae** [comment]
@JMC47 that's not really possible without providing a much bigger space for descriptions and images, imo.

**Comment by mbc07** [comment]
> Imagine a screenshot cropped down a bit of Wind Waker's sun with it on/off.
Not sure if we can include actual game screenshots without getting in trouble (especially since we're talking about Nintendo). On Play Store, for example, companion apps to Pokémon GO always show scrambled or mockup pictures on screenshots where a real Pokémon was supposed to be shown, probably to avoid copyright strikes. We might get into a similar situation...

**Comment by JMC47** [comment]
The website is full of screenshots from games?

**Comment by mbc07** [comment]
I'm not an expert on those subjects (and maybe I'm overthinking this) but the media on the website appears to fall under fair use. Unsure if that would be the case with the emulator itself, as it would be a software not affiliated with Nintendo shipping with copyrighted material.

Anyway, regardless of how this PR evolves, I wanted to point this potential issue if we end shipping game screenshots within the emulator...

**Comment by 8times9** [comment]
What if there were a "Show example" button in the description area (or something along those lines) that would open a separate window with the comparison image? That way the images are still there for users to see without cluttering the UI any more.

**Comment by spycrab** [comment]
Closing for now.
